src/Peering/Peering.Test/ScenarioTests/SetPeeringTests.cs

```diff
@@ -51,26 +51,6 @@ public void TestGetAndSetUseForPeeringService()
             TestController.NewInstance.RunPowerShellTest(this.logger, "Test-GetAndSetUseForPeeringService");
         }
 
-        /// <summary>
-        /// The test set new ip.
-        /// </summary>
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
@rileymckenna any reason the tests in this file are being removed?

Reply:
These were duplicate tests. or they aren't supported by the back end right now.